src/GitHub.Exports/Exports/ExportMetadata.cs

```diff
@@ -42,4 +44,12 @@ public interface IViewModelMetadata
     {
         UIViewType ViewType { get; }
     }
+
+    public static class ExportViewAttributeExtensions
+    {
+        public static bool IsViewType(this UserControl c, UIViewType type)
+        {
+            return c.GetType().GetCustomAttributesData().Any(x => x.AttributeType.Equals(typeof(ExportViewAttribute)) && (UIViewType)x.NamedArguments[0].TypedValue.Value == type);
```

Comment:
For types we should probably use reference equality: `x.AttributeType == typeof(ExportViewAttribute)`

Also, it seems like we have a potential `NullReferenceException` here with `x.NamedArguments` possibly being null, `x.NamedArguments[0]` possibly being null, and `x.NamedArguments[0].TypedValue` possibly being null.

If that's an impossibility, we should assert that. Otherwise we should handle the case.